### PR TITLE
SDK-294 [Node.js SDK] Sorting support

### PR DESCRIPTION
**Pull request by igor-alekseev-dev**
Pull Request Checklist
======================

Please read this for our PR Culture:
------------------------------------
https://incountry.atlassian.net/wiki/spaces/ED/pages/46858869/Pull+requests+culture+Draft

Work relates to:
----------------
Link(s) to Jira Task/User Story/Bug/Epic:
- [SDK-294](https://incountry.atlassian.net/browse/SDK-294)

Code Quality (Before submitting PR)
------------
- [x] Ensure code compiles and passes Sonar Lint.
- [x] Do your best to ensure integration tests pass. If they don't then...
- [x] Remove, ignore, or mark pending any brittle tests.
- [x] Remove commented out code.

Target Release
--------------
- [ ] Please put the target release vehicle this needs to be bundled with.
- [ ] Did you increment the version in package.json? What is the new version?

### CODE REVIEW

**Review by artemgolozubov — APPROVED** [review]
LGTM

File: src/validation/find-options.ts [hunk 1]
```diff
@@ -0,0 +1,158 @@
+import * as t from 'io-ts';
```
Comment (RommeroInCountry) [inline src/validation/find-options.ts:1]: no searching by parent_key/parentKey?

File: tests/unit/validation/find-options-tests.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  { sort: [{ a: 1 }] },
+  { sort: [{ rangeKey2: 1 }] },
+  { sort: [{ rangeKey2: 'asc', rangeKey3: 'asc' }] },
+];
+
+describe('Find Options validation', () => {
+  const codec = FindOptionsIO;
+
+
+  describe('.is()', () => {
```
Comment (RommeroInCountry) [inline tests/unit/validation/find-options-tests.ts:39]: is() ?

**Review by RommeroInCountry — CHANGES_REQUESTED** [review]
Please add README section as well

File: src/validation/find-options.ts (hunk 1 above)
Comment (igor-alekseev-dev) [reply src/validation/find-options.ts:1]: Do you mean no sorting by parentKey?

File: tests/unit/validation/find-options-tests.ts (hunk 2 above)
Comment (igor-alekseev-dev) [reply tests/unit/validation/find-options-tests.ts:39]: Yes, the method name is `is`

**Review by artemgolozubov — APPROVED** [review]
LGTM

**Review by artemgolozubov — DISMISSED** [review]
LGTM

File: README.md [hunk 3]
```diff
@@ -524,16 +524,53 @@ search (similar to the `LIKE` SQL operator, without special characters) within r
 
 #### Search options
 
-The `options` parameter defines the `limit` - number of records that are returned, and the `offset`- the starting index used for record pagination.
+The `options` parameter defines the `limit` - number of records that are returned, the `offset`- the starting index used for record pagination, and the `sort` - for the list of keys to sort the results.
```
Comment (RommeroInCountry) [inline README.md:527]: The overall tone of the sentence is kinda bulky. Let's change it this way.

The `options` parameter provides the following choices to manipulate the search results:
- `limit` - allows to limit the total number of records returned;
- `offset` - allows to specify the starting index used for records pagination;
- `sort` - allows to sort the returned records by one or multiple keys;

**Review by RommeroInCountry — CHANGES_REQUESTED** [review]

**Review by artemgolozubov — APPROVED** [review]
LGTM

**Review by RommeroInCountry — APPROVED** [review]
LGTM

**Pull request merged**